examples/historic_async.py

```diff
@@ -0,0 +1,126 @@
+import sys
+import asyncio
+import alpaca_trade_api as tradeapi
+from alpaca_trade_api.rest import TimeFrame, URL
+from alpaca_trade_api import rest_async
+from alpaca_trade_api.rest_async import gather_with_concurrency
+from alpaca_trade_api import AsyncRest
```

Comment:
this import is incorrect, `AsyncRest` is in `alpaca_trade_api.rest_async`

Reply:
I actually added this to the `__init__` file:
![image](https://user-images.githubusercontent.com/78740792/132035509-75ecdd8d-3e5d-4c3d-99ce-1075c92294d2.png)

so it's possible. but I changed the import to make it more readable.